ui-admin/src/study/participants/participantList/facets/EnrolleeSearchFacets.tsx

```diff
@@ -84,7 +84,7 @@ type FacetViewProps = {
  * Renders a facet with the appropriate component for the facet type.
  */
 export const FacetView = ({ facet, facetValue, updateValue }: FacetViewProps) => {
-  const facetType = facet.type
+  const facetType = facet.facetType
```

Comment:
It would be cool if any specified facets were expanded by default.  I believe you can do that in this file by adding a new line in the EnrolleeSearchFacets component 
```
const defaultActiveFacets = facetValues.map(facetValue =>
    facets.findIndex(facet => facetValue.facet.keyName === facet.keyName).toString())
```
and then
```
 <Accordion defaultActiveKey={defaultActiveFacets}
```